### PR TITLE
Renaming from ESA to DMA

### PR DESCRIPTION
**Pull request by esbran**
ESA to DMA name change in Readme.md

### CODE REVIEW

**Comment by esbran** [comment]
png needs to be updated

**Comment by marvinbuss** [comment]
> png needs to be updated

DONE

**Review by marvinbuss — COMMENTED** [review]
LGTM!

**Comment by marvinbuss** [comment]
I will approve and merge once all of them are ready to go.

**Review by marvinbuss — APPROVED** [review]
LGTM!
Will merge after getting the other repos ready.

File: docs/DataManagementAnalytics-Prerequisites.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Data Landing Zone - Prerequisites
 
-This template repsitory contains all templates to deploy the Data Landing Zone of the Enterprise-Scale Analytics architecture. The Data Landing Zone is a logical construct and a unit of scale in the Enterprise-Scale Analytics architecture that enables data retention and execution of data workloads for generating insights and value with data.
+This template repsitory contains all templates to deploy the Data Landing Zone of the Data Management & Analytics architecture. The Data Landing Zone is a logical construct and a unit of scale in the Data Management & Analytics architecture that enables data retention and execution of data workloads for generating insights and value with data.
```
Comment (amanjeetsingh) [inline docs/DataManagementAnalytics-Prerequisites.md:3]: Rephrase? Make it more concise?

"This template respository contains all the templates to deploy a Data Landing Zone. A Data Landing Zone is a logical construct and it's a unit of scale in the Data Management & Analytics architecture."

File: docs/DataManagementAnalytics-Prerequisites.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ To use the Deploy to Azure Button, please click on the button below:
...
+| Data Management & Analytics Scenario | Deploys a Data Management Zone and one or multiple Data Landing Zones all at once. Provides less options than the the individual Data Management Zone and Data Landing Zone deployment options. Helps you to quickly get started and make yourself familiar with the reference design. For more advanced scenarios, please deploy the artifacts individually. |[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-management-zone%2Fmain%2Fdocs%2Freference%2FdataManagementAnalytics.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-management-zone%2Fmain%2Fdocs%2Freference%2Fportal.dataManagementAnalytics.json) |
```
Comment (amanjeetsingh) [inline docs/DataManagementAnalytics-Prerequisites.md:89]: Restructure description? 

"Deploys a Data Management Zone and one or more Data Landing Zones. Goal of this reference implementation is to get users started quickly in shortest possible time and familiarise with the reference implementation. For more complex deployment scenarios, please deploy the artefacts individually as that experience is designed to give users flexibility and options to configure various aspects of a deployment."

File: README.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ The Data Management & Analytics architecture is modular by design and allows cus
...
+| Data Management & Analytics Scenario | Deploys a [Data Management Zone](https://github.com/Azure/data-management-zone) and one or multiple Data Landing Zones all at once. Provides less options than the the individual Data Management Zone and Data Landing Zone deployment options. Helps you to quickly get started and make yourself familiar with the reference design. For more advanced scenarios, please deploy the artifacts individually. |[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-management-zone%2Fmain%2Fdocs%2Freference%2FdataManagementAnalytics.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fdata-management-zone%2Fmain%2Fdocs%2Freference%2Fportal.dataManagementAnalytics.json) |  |
```
Comment (amanjeetsingh) [inline README.md:32]: "Deploys a Data Management Zone and one or more Data Landing Zones. Goal of this reference implementation is to get users started quickly in shortest possible time and familiarise with the reference implementation. For more complex deployment scenarios, please deploy the artefacts individually as that experience is designed to give users flexibility and options to configure various aspects of a deployment."

File: docs/DataManagementAnalytics-Prerequisites.md (hunk 1 above)
Comment (marvinbuss) [reply docs/DataManagementAnalytics-Prerequisites.md:3]: Please open a separate PR for these changes.

File: docs/DataManagementAnalytics-Prerequisites.md (hunk 2 above)
Comment (marvinbuss) [reply docs/DataManagementAnalytics-Prerequisites.md:89]: Please open a separate PR for these changes.

File: README.md (hunk 3 above)
Comment (marvinbuss) [reply README.md:32]: Please open a separate PR for these changes.

**Pull request merged**